irohad/consensus/yac/transport/impl/network_impl.cpp

```diff
@@ -173,6 +177,14 @@ namespace iroha {
         return grpc::Status::OK;
       }
 
+      void NetworkImpl::createPeerConnection(const model::Peer &peer) {
+        if (peers_.count(peer) == 0) {
```

Comment:
it may be a good idea to check as well does the connection is active

Reply:
I'm not sure that GRPC keeps the connection alive, so now it can just try to use the connection.